diem-move/diem-framework/experimental/Move.toml

```diff
@@ -5,7 +5,7 @@ version = "1.5.0"
 [addresses]
 Std = "0x1"
 DiemFramework = "0x1"
-BARS = "0x34CD440B72D03907100007D7BC0080A1"
+BARS = "0xB1E55ED"
```

Comment:
Wonder, why keep it the same as treasury compliance?

Reply:
I'm using treasury compliance so that in the tests we have an already created account to send mint txns, etc. If we want to use the original address (0x34CD440B72D03907100007D7BC0080A1), we will have to read the key from experimental folder and create a bars account using this key at some point during the test prep. This is because the address is derived from the key. Maybe it's not too bad or complicated. I'll explore this approach.

Reply:
This has been updated to use a non-vanity address.